### DISCUSSION TITLE
Somebody blocked me from community

### DISCUSSION THREAD

**Discussion by Saugat-Chaudhary1** [community/community#143873] [Other Feature Feedback, Questions, & Ideas]
Just testing, how to make a discussion in the community and somebody thinks himseld or herself as security officer of github and blocked me for 7 days just because I created this discussion. I wanted to delete it but there is no option to do.

**Comment by ikwerre-dev** [comment]
[To get ](https://github.com/drknzz/GitHub-Achievements)
